cmd/bootstrap/bootstrap.go

```diff
@@ -6,53 +6,117 @@ import (
 	"os"
 	"os/exec"
 	"path"
+	"path/filepath"
 
 	"github.com/skupperproject/skupper/api/types"
+	internalbundle "github.com/skupperproject/skupper/internal/nonkube/bundle"
 	"github.com/skupperproject/skupper/pkg/config"
-	"github.com/skupperproject/skupper/pkg/nonkube/apis"
+	"github.com/skupperproject/skupper/pkg/nonkube/api"
 	"github.com/skupperproject/skupper/pkg/nonkube/bundle"
 	"github.com/skupperproject/skupper/pkg/nonkube/common"
 	"github.com/skupperproject/skupper/pkg/nonkube/compat"
 	"github.com/skupperproject/skupper/pkg/nonkube/systemd"
+	"github.com/skupperproject/skupper/pkg/utils"
 	"github.com/skupperproject/skupper/pkg/version"
 )
 
 var (
-	platform = config.GetPlatform()
+	platform       = config.GetPlatform()
+	inputPath      string
+	userNamespace  string
+	bundleStrategy string
+)
+
+const (
+	description = `
+Bootstraps a nonkube Skupper site base on the provided flags.
+
+When the path (-p) flag is provided, it will be used as the source
+directory containing the Skupper custom resources to be processed,
+generating a local Skupper site using the "default" namespace, unless
+a namespace is set in the custom resources, or if the namespace (-n)
+flag is provided.
+
+A namespace is just a directory in the file system where all site specific
+files are stored, like certificates, configurations, the original sources
+(original custom resources used to bootstrap the nonkube site) and
+the runtime files generated during initialization.
+
+In case the path (-p) flag is omitted, Skupper will try to process
+custom resources stored at the sources directory of the default namespace,
+or from the namespace provided through the namespace (-n) flag.
```

Comment:
Really like the usage documentation!

The one thing that stood out to me as missing here as well as in the README is where namespaces get installed (`XDG_DATA_HOME`.)

Reply:
I agree. This is an important information to both readme and usage, I will add it.